bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/ui/view/candidate/CandidateObsController.java

```diff
@@ -3,30 +3,42 @@
 import java.beans.PropertyChangeEvent;
 import java.beans.PropertyChangeListener;
 
+import java.util.logging.Logger;
+
 import edu.gemini.qpt.core.Schedule;
 import edu.gemini.qpt.shared.sp.MiniModel;
 import edu.gemini.qpt.shared.sp.Obs;
+import edu.gemini.shared.util.immutable.ImOption;
 import edu.gemini.ui.gface.GTableController;
 import edu.gemini.ui.gface.GViewer;
 
-public class CandidateObsController implements GTableController<Schedule, Obs, CandidateObsAttribute>, PropertyChangeListener {
+public final class CandidateObsController implements GTableController<Schedule, Obs, CandidateObsAttribute>, PropertyChangeListener {
+
+    private static final Logger LOG = Logger.getLogger(CandidateObsController.class.getName());
 
     private GViewer<Schedule, Obs> viewer;
     private Obs[] obs = {};
-    
+
     public synchronized Object getSubElement(Obs obs, CandidateObsAttribute subElement) {
         switch (subElement) {
         case Dur: return obs.getRemainingTime();
         case Inst: return obs.getInstrumentString();
         case Observation: return obs;
         case P: return obs.getPriority();
-        case RA: return obs.getRa(viewer.getModel().getMiddlePoint());
+        case RA:
+            final Double ra  = ImOption.apply(viewer.getModel())
+                                       .map(s -> obs.getRa(s.getMiddlePoint()))
+                                       .getOrNull();
+            if (ra == null) {
+                LOG.warning("Prevented attempt to get middle point for a null schedule.  Returning null RA.");
+            }
+            return ra;
```

Comment:
You could complain that we're just putting off the issue for further downstream where it is less clear where the `null` comes from, but the consumer of these values seems prepared to handle `null` results.  See the `Score` `subElement` below for example.

Reply:
I won't complain 😄